src/view/range.js

```diff
@@ -82,6 +87,60 @@ export default class Range {
 	}
 
 	/**
+	 * Creates a new range with the same content, but containing all surrendering tags with no semantic meaning.
```

Comment:
I am not sure if this comment is correct, though I understand it is hard to explain how enlarge works :).

Anyway, I'd skip the second part of that because both `<p>` and `<b>` in examples have semantic meaning.

Maybe: "Creates a maximal range that has the same content as this range but is expanded in both ways (at the beginning and at the end).".